spec/values/calculation/round.hrx

```diff
@@ -0,0 +1,371 @@
+
+<===> error/syntax/no_arg/error
+Error: Missing argument $number.
+  ,--> input.scss
+1 | a {b: round()}
+  |       ^^^^^^^ invocation
+  '
+  ,--> sass:math
+1 | @function round($number) {
+  |           ============== declaration
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/two_args/compatibility/input.scss
+a {b: round(3.4px + 1px, 1 + 4)}
+
+<===> error/syntax/two_args/compatibility/error
+Error: 4.4px and 5 are incompatible.
+  ,
+1 | a {b: round(3.4px + 1px, 1 + 4)}
+  |             ^^^^^^^^^^^ 4.4px
+  |                          ===== 5
+  '
+  input.scss 1:13  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/two_args/wrong_arg/input.scss
+a {b: round(nearest, 5)}
+
+<===> error/syntax/two_args/wrong_arg/error
+Error: If strategy is not null, step is required.
+  ,
+1 | a {b: round(nearest, 5)}
+  |       ^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/strategy/number/input.scss
+a {b: round(3px, 3px, 2px)}
+
+<===> error/syntax/strategy/number/error
+Error: 3px must be either nearest, up, down or to-zero.
+  ,
+1 | a {b: round(3px, 3px, 2px)}
+  |       ^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> perserved/unknown_variable/number/input.scss
+a {b: round(test)}
+
+<===> perserved/unknown_variable/number/output.css
+a {
+  b: round(test);
+}
+
+<===>
+================================================================================
+<===> error/variable/number_and_step/input.scss
+a {b: round(4px, test, nearest)}
+
+<===> error/variable/number_and_step/error
+Error: 4px must be either nearest, up, down or to-zero.
+  ,
+1 | a {b: round(4px, test, nearest)}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> perserved/unknown_variable/two_args/input.scss
+a {b: round(test, nearest)}
+
+<===> perserved/unknown_variable/two_args/output.css
+a {
+  b: round(test, nearest);
+}
+
+<===>
+================================================================================
+<===> perserved/unknown_variable/one_arg/input.scss
+a {b: round(test, 5)}
+
+<===> perserved/unknown_variable/one_arg/output.css
+a {
+  b: round(test, 5);
+}
+
+<===>
+================================================================================
+<===> perserved/unknown_variable/step/input.scss
+a {b: round(4px, nearest)}
+
+<===> perserved/unknown_variable/step/output.css
+a {
+  b: round(4px, nearest);
+}
+
+<===>
+================================================================================
+<===> preserved/step/zero/input.scss
```

Comment:
`preserved` as a directory name should be for tests where the `round()` calculation function is preserved in the output. The test where it does simplify should omit this.

Reply:
Thanks!